ui/app/models/config-ui/message.js

```diff
@@ -72,14 +72,14 @@ export default class MessageModel extends Model {
     editType: 'textarea',
   })
   message;
-  @attr('date', {
+  @attr('dateTimeLocal', {
     editType: 'dateTimeLocal',
     label: 'Message starts',
     subText: 'Defaults to 12:00 a.m. the following day (local timezone).',
     defaultValue: addDays(startOfDay(new Date() || this.startTime), 1).toISOString(),
```

Comment:
Since `new Date()` is never falsy, this will never calculate on `this.startTime` right? Also, I think defaultValue is only used on empty values which should only be on new models, so you should be safe to remove `|| this.startTime`